libs/hwManager/constants.js

```diff
@@ -0,0 +1,31 @@
+
+export const ADD_DEVICE = 'add';
+export const REMOVE_DEVICE = 'remove';
+export const MANUFACTURERS = {
+  Ledger: {
+    name: 'Ledger',
+    models: {
+      Ledeger_Nano_S: 'Ledeger Nano S',
+      Ledeger_Nano_X: 'Ledeger Nano X',
+    },
+  },
+  Trezor: {
+    name: 'Trezor',
+    models: {
+      Trezor_Model_T: 'Trezor Model T',
+      Trezor_Model_One: 'Trezor Model One',
+    },
+  },
```

Comment:
By the architecture of putting manufacturer-specific code to each manufacturer, `MANUFACTURERS` should be IMO moved the the `ledger` folder for ledger and `trezor` folder for trezor. 

The only question is if there should be a `libs/hwManager/manufacturers/ledger/constants.js` file or if it can be part of the object in `libs/hwManager/manufacturers/ledger/index.js`

Reply:
thinking about this I will say easy way is just use a object in each index file for trezor and ledger but for consistency and thinking in more constants in future I add a constants file for each manufacture, looks more organized